en/developer_guide/media_standalone_display.rst

```diff
@@ -0,0 +1,183 @@
+Medias display
+==============
+
+Context
+-------
+
+To improve your website performances, a common way is to download the page datas from
+multiple servers. In fact, in the HTTP protocol, a browser can make only two parallel
+request to the same url.
+
+An easy way to display would be to add a new virtual host to your existing Open Orchestra
+installation. This will allow you to make two parallel request to get the page (with the
+javascript and the css) and two parallel request to the media server (to get all the images).
+
+Request scheme:
+
+IMAGE FOR THAT
+
+This solution is not ideal for a main reason : you will need to boot the whole Open Orchestra
+kernel each time you will request an image.
+
+Open Orchestra Solution
+-----------------------
+
+A simple solution would be to install only the ``open-orchestra-media-bundle`` in a single
+``Symfony`` application.
+
+Basic installation
+~~~~~~~~~~~~~~~~~~
+
+In order to install all the symfony packages, we recommand you to use the
+``open-orchestra-base-bundle``. Your ``composer.json`` file should look like:
+
+.. code-block:: json
+
+    "require": {
+        "incenteev/composer-parameter-handler": "~2.0",
+        "open-orchestra/open-orchestra-base-bundle": "*",
+        "open-orchestra/open-orchestra-media-bundle": "*"
+    },
+
+Then activate all the required bundles in your ``AppKernel.php`` file:
+
+.. code-block:: php
+
+    new Symfony\Bundle\FrameworkBundle\FrameworkBundle(),
+    new Symfony\Bundle\SecurityBundle\SecurityBundle(),
+    new Symfony\Bundle\TwigBundle\TwigBundle(),
+    new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
+
+    new Doctrine\Bundle\MongoDBBundle\DoctrineMongoDBBundle(),
+    new Knp\Bundle\GaufretteBundle\KnpGaufretteBundle(),
+
+    new OpenOrchestra\BaseBundle\OpenOrchestraBaseBundle(),
+    new OpenOrchestra\MediaBundle\OpenOrchestraMediaBundle(),
+    new OpenOrchestra\MediaModelBundle\OpenOrchestraMediaModelBundle(),
+
+It is still possible to add ``MonologBundle`` for the logging.
+
+Installation with the database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the case that you will require to request the database to extract more
+information about the ``media`` object for instance. You will need to create two
+document : ``EmbedKeyword`` and ``TranslatedValue``.
+
+Those document are only present into the ``open-orchestra-model-bundle``.
```

Comment:
Why you say that and after you say to create them inside our bundle. It's a little bit confusing